var/spack/repos/builtin/packages/gaussian-view/package.py

```diff
@@ -52,6 +62,20 @@ def url_for_version(self, version):
     def install(self, spec, prefix):
         install_tree('.', prefix)
 
+        # make sure the executable finds and uses the Spack-provided
+        # libraries, otherwise the executable may or may not run depending
+        # on what is installed on the host
+        # the $ORIGIN prefix is required for the executable to find its
+        # own bundled shared libraries
+        rpath = '$ORIGIN:$ORIGIN/lib' + ':'.join(
+            self.spec[dep].libs.directories[0]
+            for dep in ['libx11', 'libxext', 'libxrender', 'libice', 'libsm',
+                        'gl', 'glu'])
+        subprocess.call([
```

Comment:
(just a comment) Is this the Spack preferred way to call an external program? Or should we use
```
patchelf = which('patchelf')
patchelf(...)
```

The same applies to `VMD` patch.

Reply:
I wondered the same thing, but I got referred to the `intel-oneapi-*` packages as examples for this.

Reply:
Ok, since `which` just helped to discover a typo I accidentally introduced I switched over to using `which` instead of calling `subprocess.call` which failed silently.